narwhals/_duckdb/expr.py

```diff
@@ -461,6 +528,56 @@ def round(self: Self, decimals: int) -> Self:
             lambda _input: FunctionExpression("round", _input, lit(decimals)), "round"
         )
 
+    def cum_sum(self, *, reverse: bool) -> Self:
+        def func(
+            _input: duckdb.Expression,
+            partition_by: Sequence[str],
+            order_by: Sequence[str],
+        ) -> duckdb.Expression:
+            if reverse:
+                order_by_sql = "order by " + ", ".join(
+                    f'"{x}" desc nulls last' for x in order_by
```

Comment:
@MarcoGorelli I commented this on the stream, not sure it was ever tried out, anyways now in context 🙂 
```suggestion
                    f'{x!r} desc nulls last' for x in order_by
```

- https://docs.python.org/3/reference/lexical_analysis.html#formatted-string-literals

Reply:
i don't think that's the same, we need double quotes around the column names as that's what sql expects (single quotes are string literals)

Reply:
> i don't think that's the same, we need double quotes around the column names as that's what sql expects (single quotes are string literals)

Does it not respect the outer quotes?

I would've thought these two used different internal ones:

```py
f'{x!r} desc nulls last' for x in order_by
f"{x!r} desc nulls last" for x in order_by
```